docs/cli/coder.md

```diff
@@ -0,0 +1,72 @@
+## coder
+
+### Synopsis
+
+Coder v0.0.0-devel — A tool for provisioning self-hosted development environments with Terraform.
```

Comment:
Just double-check: is `v0.0.0-devel` fine for us, or is it better to replace it with something meaningful or skip it?

Reply:
I think this is something to fix but I didn't figure out a good solution for it 🤔. Maybe I should just remove the `v0.0.0-devel` while removing the ANSI character? Ugly solution but works.

Reply:
It is not ugly as I thought haha. Going to merge this but feel free to comment on any improvements.